### PR TITLE
add pagination to form instances

### PR DESCRIPTION
**Pull request by Rahien**
image

### CODE REVIEW

File: app/components/form/instance-table.js [hunk 1]
```diff
@@ -41,16 +41,45 @@ export default class InstanceTableComponent extends Component {
     }
   }
 
+  @action
   async onInit() {
```
Comment (bfidlers) [inline app/components/form/instance-table.js:45]: This seems more complicated than it should be, I think you could rely more on ember to track the whole pagination etc. but I don't really have time to investigate it now.

Comment (bfidlers) [reply app/components/form/instance-table.js:45]: or maybe not since we don't use resources, I'll have a look at that on Friday.

Comment (Rahien) [reply app/components/form/instance-table.js:45]: sadly no, the data table assumes the same meta extensions exist like they were done for ember data relations/queries

**Review by bfidlers — APPROVED** [review]
Bit sad about the manual pagination, but unfortunately that is a result of not being able to use ember resources.

Looks good otherwise.

**Pull request merged**